lang/en.json

```diff
@@ -824,6 +824,8 @@
   "expense.paymentProcessorFeeInCollectiveCurrency": "payment processor fee",
   "expense.payoutMethod": "payout method",
   "Expense.PayTo": "Pay to",
+  "Expense.PayFrom": "Pay to",
```

Comment:
@RichardLitt : This line "Expense.PayFrom": "Pay to" is not needed I think as it's not used. Overall I don't think you need to update the lang files manually. All you got to do is run `npm start langs:update` which will auto update all the language files. Whether it does the translation accurately, I don't think so; but my understanding is that at certain time intervals the @opencollective-robot does the translations automatically (correct me if I am wrong @Betree). 🤔

Reply:
Ah, yikes. This line shouldn't be here. My bad!

Reply:
@SudharakaP Corrrect, we should not edit the `**.json` files manually. All you got to do after editing the source string is run `npm run langs:update` to refresh them all, then @opencollective-robot (linked to Crowdin) will update them with the translations.

@RichardLitt Let me know if you don't have the bandwidth to do this, we can take care of it.

Reply:
Done!